### PR TITLE
Debian package building

### PR DESCRIPTION
**Pull request by ervanalb**
This pull request migrates Travis to Bionic and gets it building a .deb file.
Once Debian packages are built by Travis, we can put them on a PPA through launchpad.

Related to #47 and #94

### CODE REVIEW

**Comment by ervanalb** [comment]
Also related to #94

File: deploy/debian.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+if [ -z "$3" ]; then
+    echo "You must specify an upstream version (e.g. 0.6.1) as the third argument" >&2
+    exit 1
+fi
+
+if [ -z "$4" ]; then
+    echo "You must specify a packaging version (e.g. 1) as the fourth argument" >&2
+    exit 1
+fi
```
Comment (zbanks) [inline deploy/debian.sh]: `if [ "$#" -ne 4 ]`?

File: .travis/after_success.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -ex
+
+if [[ $TRAVIS_OS_NAME == 'osx' ]]
+then
+    make bundle
+elif [[ $TRAVIS_OS_NAME == 'linux' ]]
```
Comment (zbanks) [inline .travis/after_success.sh]: Can this be pushed into the `yml`? `on.condition: "$TRAVIS_OS_NAME == linux"` or does that not work

File: deploy/debian.sh (hunk 1 above)
Comment (ervanalb) [reply deploy/debian.sh]: 4 or 5 arguments is OK

File: .travis/after_success.sh (hunk 2 above)
Comment (ervanalb) [reply .travis/after_success.sh]: This was the pattern already in most of the travis scripts, I don't really feel like changing it. TBH I don't understand travis's yml.

**Pull request merged**